lib/analysis/user.py

```diff
@@ -314,7 +316,7 @@ def build_centroid(km):
 
     stop_words_extended = extended_stop_words(nicks_for_stop_words, stop_word_without_apostrophe) 
     
-    vectorizer = TfidfVectorizer(max_df=1, min_df=0.5, stop_words=stop_words_extended,
+    vectorizer = TfidfVectorizer(max_df=0.5, min_df=2, stop_words=stop_words_extended,
```

Comment:
@rohangoel96 please throw some light on the need and justification for max_df and min_df parameters.

Reply:
The values were actually changed in the previous commit. I thought that since changing the values now would change the results too, I reverted them back in this commit.

Reply:
max_df = ignore words if present in more than 'max_df'% of documents
min_df = ignore words if freq less than 'min_df'

Reply:
Based on your explanation, it looks like we are ignoring words that appear in more than 2% of the documents and also ignoring the words that appear in less than 0.5% of the documents. Is that a correction understanding?

Is there any disadvantage to making max_df=100 and min_df=0 ?